### ISSUE TITLE
Implementing In Game Currency

### ISSUE THREAD

**Issue by ydaniels** [4ian/GD#349]
Hello nice platform but does this platform support in game currency

**Comment by VenHayz** [comment]
Do you mean in-app purchases? In-app purchases allow you to sell things in your game to the player if they pay a real dollar amount inside your game when prompted. Is this what you're looking for?

**Comment by ydaniels** [comment]
Yep a feature of the platform allows you to easily sell stuffs that can be used in the game itself .
e.g when a player pays they get extra life or get to run faster

**Comment by VenHayz** [comment]
@victorlevasseur does the game engine have an implementation for in-app purchases on platforms like iOS and/or Android? As asked by @ydaniels

**Comment by 4ian** [comment]
No such extension is available for now! Help is welcome if anyone is interested in developing it 😄

**Comment by blurymind** [comment]
you would have to implement support for google's in app purchase services
if you ever want to make profitable freemium games mobile games with
gdevelop
=)

**Comment by ydaniels** [comment]
@blurymind I think it is better to develop it directly on the platform than integrating  it with cocos2d but using this [http://www.soom.la/](Soomla) framework which cocos2dx-store is based on should  be easy
@4ian As for the development is their an exposed api in js one can use to develop it faster

**Comment by arthuro555** [comment]
With #1717, this should be pretty easy to implement using https://github.com/j3k0/cordova-plugin-purchase. I recommend removing the "Careful thinking/design or refactoring needed" tag for that reason.
